compat/src/index.d.ts

```diff
@@ -46,6 +46,7 @@ declare namespace React {
 	export import RefObject = preact.RefObject;
 	export import Component = preact.Component;
 	export import FunctionComponent = preact.FunctionComponent;
+	export import preact.ComponentType;
```

Comment:
Oops, sorry. Mistyped this

```suggestion
	export import ComponentType = preact.ComponentType;
```